### PR TITLE
WIP Look for strings in correct bundle

### PR DESCRIPTION
**Pull request by jballance**
TLDR: Had to tell the app where to look for the strings and the changes allowed us to put the strings files back in the right spot

**exporting locally is printing out the `Resources/.lproj/` path but not on build server?**

The SRLoc method used before would tell the app to look in the right bundle, but we stopped using that and used NSLocalizedString macro so that the strings would get exported in the xliff. But then the app started looking in the main bundle resources for the strings file, instead of the SR framework. I found out that we could go back to using SRLoc to fix the bundle issue, and then add it to a list of macros in the build settings so that the strings would still get exported properly. (I didn't know that was a thing before). After doing that, I moved the strings files back to the Resources dir for kicks & giggles and exporting the xliffs actually had the right path. 

So the last many weeks of changes could have been avoided by adding this simple macro to a list...

The string that was showing in English before this work:

Adding SRLoc to the list of localization macros in the project settings

The path in the xliff now

### CODE REVIEW

**Comment by jballance** [comment]
this didn't work as well as I thought it would. when using the SRLoc macro, the xcodebuild tool gives us some error about the macro. it continues to export the strings from the existing en strings file into the xliff, but it doesn't seem like it's actually exporting the strings in the code. if you delete the en strings file, the xliff ends up empty. 
adding a second argument to the macro somehow silenced the error from xcodebuild, but then the file path in the xliff was "Localizable.strings" and not "ShortcutRecorder" (second arg was not counted as the table name like I hoped)
